simcon_project/users/views/create_assignment.py

```diff
@@ -0,0 +1,144 @@
+from django.shortcuts import render
+from users.models import SubjectLabel, Assignment, Student, Researcher
+from conversation_templates.models import ConversationTemplate
+from django.views.generic import TemplateView
+from django.core import serializers
+from django.http import HttpResponse
+from django.core.mail import send_mail
+from apscheduler.schedulers.background import BackgroundScheduler
+from tzlocal import get_localzone
+import json
+import datetime
+
+
+class CreateAssignmentView(TemplateView):
```

Comment:
Make sure only a researcher can access this view. Maybe look at how is_researhcer(user) from researcher_home.py is used in other views.